packages/frontend/src/components/Explorer/VisualizationCard/VisualizationCard.tsx

```diff
@@ -62,6 +65,7 @@ const VisualizationCard: FC = memo(() => {
     );
 
     const { data: explore } = useExplore(unsavedChartVersion.tableName);
+    const { projectUuid } = useParams<{ projectUuid: string }>();
```

Comment:
I don't think we should be using useParams here, if we include this component in a page that does not have `projectUuid` then this will break. 
For shared components we always pass it as an argument .